noobaa_deploy_k8s.sh cannot set value for 'jwt' in noobaa-secrets

### Environment info
- NooBaa Version: **5.0.1**
- Platform: **OpenShift 3.11**

### Actual behavior
1.Use noobaa_deploy_k8s.sh to deploy noobaa on OpenShift, there is no value for 'jwt' in  noobaa-secrets. That causes noobaa pod cannot be ready.

### Expected behavior
1.NooBaa pod can be ready

### Steps to reproduce
1. $ ./noobaa_deploy_k8s.sh deploy -n noobaa

### More information - Screenshots / Logs / Other output
Aug-21 7:51:31.268 [/8868] [ERROR] CONSOLE:: PANIC: process uncaughtException Error: JWT_SECRET env variable not found. it must exist when running in kuberentes
    at get_jwt_secret (/root/node_modules/noobaa-core/src/server/common_services/auth_server.js:664:19)
    at Object.<anonymous> (/root/node_modules/noobaa-core/src/server/common_services/auth_server.js:22:20)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/root/node_modules/noobaa-core/src/server/object_services/map_server.js:17:21)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Module.require (internal/modules/cjs/loader.js:636:17)
2019-08-21 07:51:31,283 INFO exited: bg_workers (exit status 1; not expected)

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "noobaa-core",
-  "version": "5.1.0",
+  "version": "5.1.1",
   "private": true,
   "license": "SEE LICENSE IN LICENSE",
   "description": "",

--- a/src/deploy/NVA_build/noobaa_deploy_k8s.sh
+++ b/src/deploy/NVA_build/noobaa_deploy_k8s.sh
@@ -410,7 +410,7 @@ function create_cred_secret {
 
 function create_noobaa_secrets {
     local SERVER_SECRET=$(openssl rand -hex 4)
-    local JWT_SECRET=$(openssl rand -base64 20 | openssl sha512 -hmac | cut -c10-44)
+    local JWT_SECRET=$(openssl rand -hex 20)
     ${KUBECTL} delete secret ${NOOBAA_SECRETS_NAME} &> /dev/null
     ${KUBECTL} create secret generic ${NOOBAA_SECRETS_NAME} \
         --from-literal=server_secret=${SERVER_SECRET} \

